HyperV/scripts/create-environment.ps1

```diff
@@ -0,0 +1,277 @@
+    }
+    Catch
+    {
+        Throw "Vrtualenv already exists. Environment not clean."
+    }
+}
+
+Write-Host "Cleaning up the config folder."
+if ($hasConfigDir -eq $false) {
+    mkdir $configDir
+}else{
+    Try
+    {
+        Remove-Item -Recurse -Force $configDir\*
+    }
+    Catch
+    {
+        Throw "Can not clean the config folder"
+    }
+}
+
+if ($hasProject -eq $false){
+    Throw "$projectName repository was not found. Please run gerrit-git-pref for this project first"
+}
+
+if ($hasBinDir -eq $false){
+    mkdir $binDir
+}
+
+if (($hasMkisoFs -eq $false) -or ($hasQemuImg -eq $false)){
+    Invoke-WebRequest -Uri "http://dl.openstack.tld/openstack_bin.zip" -OutFile "$bindir\openstack_bin.zip"
+    if (Test-Path "C:\Program Files\7-Zip\7z.exe"){
+        pushd $bindir
+        & "C:\Program Files\7-Zip\7z.exe" x -y "$bindir\openstack_bin.zip"
+        Remove-Item -Force "$bindir\openstack_bin.zip"
+        popd
+    } else {
+        Throw "Required binary files (mkisofs, qemuimg etc.)  are missing"
+    }
+}
+
+if ($hasVirtualenv -eq $true){
+    Throw "Vrtualenv already exists. Environment not clean."
+}
+
+if ($hasNovaTemplate -eq $false){
+    Throw "Nova template not found"
+}
+
+if ($hasNeutronTemplate -eq $false){
+    Throw "Neutron template not found"
+}
+
+git config --global user.email "hyper-v_ci@microsoft.com"
+git config --global user.name "Hyper-V CI"
+
+
+if ($buildFor -eq "openstack/nova"){
+    ExecRetry {
+        GitClonePull "$buildDir\neutron" "https://github.com/openstack/neutron.git" $branchName
+    }
+    ExecRetry {
+        GitClonePull "$buildDir\networking-hyperv" "https://github.com/stackforge/networking-hyperv.git" "master"
+    }
+}elseif ($buildFor -eq "openstack/neutron" -or $buildFor -eq "openstack/quantum"){
+    ExecRetry {
+        GitClonePull "$buildDir\nova" "https://github.com/openstack/nova.git" $branchName
+    }
+    ExecRetry {
+        GitClonePull "$buildDir\networking-hyperv" "https://github.com/stackforge/networking-hyperv.git" "master"
+    }
+}elseif ($buildFor -eq "stackforge/networking-hyperv"){
+    ExecRetry {
+        GitClonePull "$buildDir\nova" "https://github.com/openstack/nova.git" $branchName
+    }
+    ExecRetry {
+        GitClonePull "$buildDir\neutron" "https://github.com/openstack/neutron.git" $branchName
+    }
+}else{
+    Throw "Cannot build for project: $buildFor"
+}
+
+$hasLogDir = Test-Path $remoteLogs\$hostname
+if ($hasLogDir -eq $false){
+    mkdir $remoteLogs\$hostname
+}
+
+$hasConfigDir = Test-Path $remoteConfigs\$hostname
+if ($hasConfigDir -eq $false){
+    mkdir $remoteConfigs\$hostname
+}
+
+cmd.exe /C virtualenv --system-site-packages $virtualenv
+
+if ($? -eq $false){
+    Throw "Failed to create virtualenv"
+}
+
+cp $templateDir\distutils.cfg $virtualenv\Lib\distutils\distutils.cfg
+
+# Hack due to cicso patch problem:
+$missingPath="C:\Openstack\build\openstack\neutron\etc\neutron\plugins\cisco\cisco_cfg_agent.ini"

```

Comment:
This used to be required long time ago, shold check if it's still the case now, instead of just including it here